sourmash/sbt.py

```diff
@@ -327,9 +328,10 @@ def find(self, search_fn, *args, **kwargs):
                     elif isinstance(node_g, Node):
                         if kwargs.get('dfs', True):  # defaults search to dfs
                             for c in self.children(node_p):
-                                queue.insert(0, c.pos)
+                                if c.node:
```

Comment:
could you explain/comment these changes, pls?

Reply:
the node was being inserted in the search queue even if the node data was empty, which is a small difference but avoids checking it in the next iteration. Now it is only inserted in the queue if the node actually exists (and is not `None`)

Reply:
👍